### PR TITLE
add Web App Manifest and favicons

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: manifest.webmanifest [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  "name": "A-Painter",
+  "short_name": "A-Painter",
+  "description": "Paint in VR in your browser. Powered by A-Frame.",
+  "start_url": "./?src=manifest",
+  "display": "fullscreen",
+  "theme_color": "#0a0a0a",
+  "background_color": "#0a0a0a",
+  "icons": [
+    {
+      "src": "/img/icon-512.png",
```
Comment (cvan) [inline manifest.webmanifest]: let me know if you have a better icon to use (possibly the `A` wordmark from the A-Frame splash screen)

File: manifest.webmanifest [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  "lang": "en",
+  "dir": "ltr",
+  "name": "A-Painter",
+  "short_name": "A-Painter",
+  "description": "Paint in VR in your browser. Powered by A-Frame.",
+  "start_url": "./?src=manifest",
+  "display": "fullscreen",
+  "theme_color": "#0a0a0a",
+  "background_color": "#0a0a0a",
+  "icons": [
```
Comment (cvan) [inline manifest.webmanifest]: FYI: I compressed these PNGs using `pngquant`

**Comment by fernandojsg** [comment]
@cvan thanks! I'll just merge this and @feiss could just open another PR when he'll have the icon ready ^_^

**Comment by cvan** [comment]
thanks!

**Pull request merged**